src/quicr_server_raw_session.cpp

```diff
@@ -111,26 +150,31 @@ QuicRServerRawSession::run()
 void
 QuicRServerRawSession::publishIntentResponse(
   const quicr::Namespace& quicr_namespace,
+  const qtransport::TransportContextId& context_id,
   const PublishIntentResult& result)
 {
-  if (!publish_namespaces.count(quicr_namespace))
+  if (!publish_intent_state.count(context_id))
```

Comment:
The reason I kept it is because the way I am using publish_state, the publish intent context could be overridden by the next publish object. Though maybe it's a non-issue, since I think I changed it so that the publish namespace isn't added until a successful publishIntentResponse (effectively blocking published named objects going through yet)... so might be safe now. I'll poke around with that again.